library/extractor/src/main/java/com/google/android/exoplayer2/metadata/id3/TextInformationFrame.java

```diff
@@ -19,52 +19,77 @@
 
 import android.os.Parcel;
 import android.os.Parcelable;
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import com.google.android.exoplayer2.MediaMetadata;
 import com.google.android.exoplayer2.util.Util;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 /** Text information ID3 frame. */
 public final class TextInformationFrame extends Id3Frame {
+  private final static String MULTI_VALUE_DELIMITER = ", ";
```

Comment:
I would just in-line this, i think it makes the usages clearer

Reply:
Resolved in 157f1f0.